Source/Core/VideoCommon/TextureUtils.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "VideoCommon/TextureUtils.h"
+
+#include <fmt/format.h>
+
+#include "Common/FileUtil.h"
+#include "Core/Config/GraphicsSettings.h"
+#include "Core/ConfigManager.h"
+
+#include "VideoCommon/AbstractTexture.h"
+
+namespace VideoCommon::TextureUtils
+{
+void DumpTexture(const ::AbstractTexture& texture, std::string basename, unsigned int level,
```

Comment:
since this is being moved along with some modifications, maybe also convert `unsigned int` into `u32` or `uint32_t` for brevity